### PR TITLE
[onert] add shape inference interface and its test case for SQUARED_DIFF op.

### PR DESCRIPTION
**Pull request by Hyunjun85**
Add shape inference interface and its tc.

ONE-DCO-1.0-Signed-off-by: H Kim <hyunjun2.kim@samsung.com>

### CODE REVIEW

**Comment by dr-venkman** [comment]
Perhaps include positive and (if applicable) negative testcases for SquaredDifference under `test/util/ShapeInference.cc`?

File: runtime/onert/core/src/util/shapeinf/SquaredDifference.cc [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#include "util/ShapeInference.h"
+
+namespace onert
+{
+namespace shape_inference
+{
+
+void StaticInferer::visit(const ir::operation::SquaredDifference &op)
+{
+  handleBinaryArithmeticOp(op, op.getInputs().at(ir::operation::SquaredDifference::Input::LHS),
```
Comment (btheosam) [inline runtime/onert/core/src/util/shapeinf/SquaredDifference.cc:26]: Is it already existing?

File: runtime/onert/core/src/util/shapeinf/SquaredDifference.cc [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ */
+
+#include "util/ShapeInference.h"
+
+namespace onert
+{
+namespace shape_inference
+{
+
+void StaticInferer::visit(const ir::operation::SquaredDifference &op)
```
Comment (btheosam) [inline runtime/onert/core/src/util/shapeinf/SquaredDifference.cc:24]: Is it to infer the shape of the output of squared difference between two tensors? If so should n't it be unary operand?

File: tests/nnapi/specs/Ex/squared_difference_ex_dynamic_nnfw.mod.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+test_node_input = dynamic_layer.getTestNodeInput()
+
+i2 = Input("op2", "TENSOR_FLOAT32","{2, 2}")
+t1 = Internal("op3", "TENSOR_FLOAT32", "{}")
+o1 = Output("op3", "TENSOR_FLOAT32", "{3, 2, 2, 2}")
+
+model = model.Operation("SQUARED_DIFFERENCE_EX", test_node_input, i2).To(t1)
+model = model.Operation("SQUARED_DIFFERENCE_EX", t1, i2).To(o1)
+# model = model.Operation("SQUARED_DIFFERENCE_EX", test_node_input, i2).To(o1)
```
Comment (ragmani) [inline tests/nnapi/specs/Ex/squared_difference_ex_dynamic_nnfw.mod.py:19]: Why don't you remove this?

Comment (Hyunjun85) [reply tests/nnapi/specs/Ex/squared_difference_ex_dynamic_nnfw.mod.py]: Sorry, I missed it. 
It was emoved.

**Pull request merged**